README.md

```diff
@@ -32,6 +32,10 @@ it, simply add the following line to your Podfile:
 pod "SimpleKeychain"
 ```
 
+Then, run `pod install`.
+
+> For further reference on Cocoapods, check [their official documentation](http://guides.cocoapods.org/using/getting-started.html).
```

Comment:
> For further reference on

I would change this to "For more information on/about.." to be consistent with line 49, and I think it reads better.

```suggestion
> For more information on Cocoapods, check [their official documentation](http://guides.cocoapods.org/using/getting-started.html).
```